src/features/types/index.ts

```diff
@@ -127,6 +127,9 @@ export type TFlexBasis = InferSide<ClassNames, 'flex-basis'>
 export type TContainerType = InferSide<ClassNames, 'container'>
 export type TVisible = InferSide<ClassNames, 'visible'>
 
+//clipath banner color
+export type TClipPathBannerColor = InferSide<ClassNames, 'clip-path-product'>
```

Comment:
added 
| 'clip-path-product-brand'
| 'clip-path-product-ctrade'
to classname.d.ts